cockroachcloud/cluster-management.md

```diff
@@ -179,6 +181,19 @@ To restore your data, [contact us](https://support.cockroachlabs.com).
 
 Additionally, you can [backup and restore](../{{site.current_cloud_version}}/take-full-and-incremental-backups.html) data on your own.
 
+## Configure PCI compliant features (Dedicated advanced)
+
+{{ site.data.products.dedicated }} advanced clusters have a **PCI ready** panel to monitor the status of security features required for PCI compliance. Feature statuses will update from **INACTIVE** to **ACTIVE** once you configure them. Learn more about configuring these features:
+
+- [Audit logs](cloud-org-audit-logs.html)
+- [Customer-Managed Encryption Keys (CMEK)](managing-cmek.html)
+- [Egress Perimeter Controls](egress-perimeter-controls.html)
+- Single Sign-On (SSO) for both your [{{ site.data.products.db }} organization](configure-cloud-org-sso.html) and the [DB Console](../{{site.versions["stable"]}}/sso-db-console.html)
```

Comment:
SSO should be at the org level and the cluster level. Is the DB Console the cluster level one or is that a different SSO? I am not sure how we refer to it for the cluster level.